fs/fs_zenfs.cc

```diff
@@ -417,6 +417,8 @@ IOStatus ZenFS::DeleteFile(std::string fname) {
     } else {
       zoneFile.reset();
     }
+  } else {
+    s = IOStatus::NotFound("ZenfFS::DeleteFile(): File not found");
```

Comment:
Typo: ZenfFS -> ZenFS.
Could we log the filename which was not found, will be useful in debugging issues.

Reply:
Thanks @aravind-wdc. Pushed a fix.